python/ray/includes/libcoreworker.pxd

```diff
@@ -124,7 +124,7 @@ cdef extern from "ray/core_worker/core_worker.h" nogil:
         CRayStatus RemovePlacementGroup(
             const CPlacementGroupID &placement_group_id)
         CRayStatus WaitPlacementGroupReady(
-            const CPlacementGroupID &placement_group_id, int timeout_seconds)
+            const CPlacementGroupID &placement_group_id, int64_t timeout_seconds)
```

Comment:
converted it to int64_t to avoid overflow (I believe there could be overflow if we do implicit conversion from int -> int64_t. Lmk if I am wrong)

Reply:
huh i don't think so. https://en.cppreference.com/w/c/language/conversion